examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/appinfo/AppInfoScreen.kt

```diff
@@ -64,6 +71,16 @@ fun AppInfoScreen(viewModel: AppInfoScreenViewModel = viewModel<AppInfoScreenVie
         isVisible = isDebugBottomSheetVisible,
         onDismissCallback = { isDebugBottomSheetVisible = false },
     )
+
+    // CustomerCenter WIP: Uncomment once access to CustomerCenter is public.
```

Comment:
I need to keep this commented out since I'm not making public the composables in the revenuecatui module yet. I'm planning to create a PR just making those public and this available so we can cherry-pick as we did in iOS. Lmk if you have a better idea!

Reply:
Short term I don't have a better idea. In the medium term, maybe we can do something with variants, e.g. create a customer center variant that has a different source set. Or maybe we can see if we can replicate the compiler flag approach in iOS 🤔

Reply:
Hmm yeah, I don't think there is a way to have a compiler flag in Android afaik? As for a variant, it could work but I think it's a lot of overhead IMO for this?

Reply:
Yea that's probably a lot of overhead indeed. 😕